pavlo976/BuildMinder-App
# Feature/querying data

### What changed?
React Query was added to entire app
### How to test it?
In any component, try to fetch data from some API via useQuery function and log to the console if data is received and open react-query-devtools (bottom left corner of the browser) to check that the data is displayed correctly
### What's missing?
Everything is up to date
